[2024-02-07T08:48:37.307+0000] {processor.py:161} INFO - Started process (PID=605) to work on /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:48:37.307+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/10_emp_details.py for tasks to queue
[2024-02-07T08:48:37.309+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:48:37.309+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:48:37.314+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:48:37.313+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/10_emp_details.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/10_emp_details.py", line 19, in <module>
    import Task_builder
ModuleNotFoundError: No module named 'Task_builder'
[2024-02-07T08:48:37.314+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:48:37.338+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/10_emp_details.py took 0.035 seconds
[2024-02-07T08:49:07.396+0000] {processor.py:161} INFO - Started process (PID=671) to work on /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:49:07.397+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/10_emp_details.py for tasks to queue
[2024-02-07T08:49:07.399+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:49:07.399+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:49:07.402+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:49:07.402+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/10_emp_details.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/10_emp_details.py", line 19, in <module>
    import Task_builder
ModuleNotFoundError: No module named 'Task_builder'
[2024-02-07T08:49:07.403+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:49:07.424+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/10_emp_details.py took 0.031 seconds
[2024-02-07T08:49:37.734+0000] {processor.py:161} INFO - Started process (PID=737) to work on /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:49:37.736+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/10_emp_details.py for tasks to queue
[2024-02-07T08:49:37.738+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:49:37.738+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:49:37.741+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:49:37.740+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/10_emp_details.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/10_emp_details.py", line 19, in <module>
    import Task_builder
ModuleNotFoundError: No module named 'Task_builder'
[2024-02-07T08:49:37.742+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:49:37.767+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/10_emp_details.py took 0.040 seconds
[2024-02-07T08:50:07.840+0000] {processor.py:161} INFO - Started process (PID=803) to work on /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:50:07.841+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/10_emp_details.py for tasks to queue
[2024-02-07T08:50:07.844+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:50:07.843+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:50:07.847+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:50:07.846+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/10_emp_details.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/10_emp_details.py", line 19, in <module>
    import Task_builder
ModuleNotFoundError: No module named 'Task_builder'
[2024-02-07T08:50:07.847+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:50:07.872+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/10_emp_details.py took 0.034 seconds
[2024-02-07T08:50:38.650+0000] {processor.py:161} INFO - Started process (PID=869) to work on /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:50:38.652+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/10_emp_details.py for tasks to queue
[2024-02-07T08:50:38.656+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:50:38.655+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:50:38.660+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:50:38.659+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/10_emp_details.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/10_emp_details.py", line 19, in <module>
    import Task_builder
ModuleNotFoundError: No module named 'Task_builder'
[2024-02-07T08:50:38.661+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:50:38.684+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/10_emp_details.py took 0.036 seconds
[2024-02-07T08:51:09.326+0000] {processor.py:161} INFO - Started process (PID=935) to work on /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:51:09.327+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/10_emp_details.py for tasks to queue
[2024-02-07T08:51:09.329+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:51:09.328+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:51:09.331+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:51:09.331+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/10_emp_details.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/10_emp_details.py", line 19, in <module>
    import Task_builder
ModuleNotFoundError: No module named 'Task_builder'
[2024-02-07T08:51:09.332+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:51:09.350+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/10_emp_details.py took 0.026 seconds
[2024-02-07T08:51:40.363+0000] {processor.py:161} INFO - Started process (PID=1001) to work on /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:51:40.364+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/10_emp_details.py for tasks to queue
[2024-02-07T08:51:40.366+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:51:40.365+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:51:40.369+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:51:40.368+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/10_emp_details.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/10_emp_details.py", line 19, in <module>
    import Task_builder
ModuleNotFoundError: No module named 'Task_builder'
[2024-02-07T08:51:40.369+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:51:40.388+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/10_emp_details.py took 0.027 seconds
[2024-02-07T08:52:10.626+0000] {processor.py:161} INFO - Started process (PID=1067) to work on /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:52:10.627+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/10_emp_details.py for tasks to queue
[2024-02-07T08:52:10.628+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:52:10.628+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:52:10.631+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:52:10.630+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/10_emp_details.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/10_emp_details.py", line 19, in <module>
    import Task_builder
ModuleNotFoundError: No module named 'Task_builder'
[2024-02-07T08:52:10.631+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:52:10.657+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/10_emp_details.py took 0.034 seconds
[2024-02-07T08:52:40.716+0000] {processor.py:161} INFO - Started process (PID=1133) to work on /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:52:40.717+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/10_emp_details.py for tasks to queue
[2024-02-07T08:52:40.720+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:52:40.719+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:52:40.723+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:52:40.722+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/10_emp_details.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/10_emp_details.py", line 19, in <module>
    import Task_builder
ModuleNotFoundError: No module named 'Task_builder'
[2024-02-07T08:52:40.723+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:52:40.742+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/10_emp_details.py took 0.029 seconds
[2024-02-07T08:53:11.596+0000] {processor.py:161} INFO - Started process (PID=1199) to work on /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:53:11.597+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/10_emp_details.py for tasks to queue
[2024-02-07T08:53:11.599+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:53:11.598+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:53:11.601+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:53:11.601+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/10_emp_details.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/10_emp_details.py", line 19, in <module>
    import Task_builder
ModuleNotFoundError: No module named 'Task_builder'
[2024-02-07T08:53:11.602+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:53:11.620+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/10_emp_details.py took 0.027 seconds
[2024-02-07T08:53:41.885+0000] {processor.py:161} INFO - Started process (PID=1265) to work on /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:53:41.887+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/10_emp_details.py for tasks to queue
[2024-02-07T08:53:41.889+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:53:41.888+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:53:41.892+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:53:41.891+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/10_emp_details.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/10_emp_details.py", line 19, in <module>
    import Task_builder
ModuleNotFoundError: No module named 'Task_builder'
[2024-02-07T08:53:41.892+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:53:41.911+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/10_emp_details.py took 0.028 seconds
[2024-02-07T08:54:12.940+0000] {processor.py:161} INFO - Started process (PID=1331) to work on /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:54:12.942+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/10_emp_details.py for tasks to queue
[2024-02-07T08:54:12.943+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:54:12.943+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:54:12.946+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:54:12.946+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/10_emp_details.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/10_emp_details.py", line 19, in <module>
    import Task_builder
ModuleNotFoundError: No module named 'Task_builder'
[2024-02-07T08:54:12.947+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:54:12.965+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/10_emp_details.py took 0.027 seconds
[2024-02-07T08:54:43.944+0000] {processor.py:161} INFO - Started process (PID=1397) to work on /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:54:43.945+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/10_emp_details.py for tasks to queue
[2024-02-07T08:54:43.947+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:54:43.947+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:54:43.950+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:54:43.949+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/10_emp_details.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/10_emp_details.py", line 19, in <module>
    import Task_builder
ModuleNotFoundError: No module named 'Task_builder'
[2024-02-07T08:54:43.950+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:54:43.969+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/10_emp_details.py took 0.029 seconds
[2024-02-07T08:55:14.242+0000] {processor.py:161} INFO - Started process (PID=1463) to work on /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:55:14.244+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/10_emp_details.py for tasks to queue
[2024-02-07T08:55:14.247+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:55:14.246+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:55:14.251+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:55:14.251+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/10_emp_details.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/10_emp_details.py", line 19, in <module>
    import Task_builder
ModuleNotFoundError: No module named 'Task_builder'
[2024-02-07T08:55:14.252+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:55:14.270+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/10_emp_details.py took 0.032 seconds
[2024-02-07T08:55:44.345+0000] {processor.py:161} INFO - Started process (PID=1529) to work on /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:55:44.347+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/10_emp_details.py for tasks to queue
[2024-02-07T08:55:44.349+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:55:44.348+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:55:44.352+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:55:44.351+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/10_emp_details.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/10_emp_details.py", line 19, in <module>
    import Task_builder
ModuleNotFoundError: No module named 'Task_builder'
[2024-02-07T08:55:44.352+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:55:44.374+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/10_emp_details.py took 0.033 seconds
[2024-02-07T08:56:14.738+0000] {processor.py:161} INFO - Started process (PID=1595) to work on /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:56:14.739+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/10_emp_details.py for tasks to queue
[2024-02-07T08:56:14.741+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:56:14.741+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:56:14.745+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:56:14.744+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/10_emp_details.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/10_emp_details.py", line 19, in <module>
    import Task_builder
ModuleNotFoundError: No module named 'Task_builder'
[2024-02-07T08:56:14.746+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:56:14.766+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/10_emp_details.py took 0.031 seconds
[2024-02-07T08:56:45.014+0000] {processor.py:161} INFO - Started process (PID=1662) to work on /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:56:45.015+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/10_emp_details.py for tasks to queue
[2024-02-07T08:56:45.017+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:56:45.017+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:56:45.021+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:56:45.020+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/10_emp_details.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/10_emp_details.py", line 19, in <module>
    import Task_builder
ModuleNotFoundError: No module named 'Task_builder'
[2024-02-07T08:56:45.021+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:56:45.041+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/10_emp_details.py took 0.030 seconds
[2024-02-07T08:57:15.221+0000] {processor.py:161} INFO - Started process (PID=1728) to work on /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:57:15.223+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/10_emp_details.py for tasks to queue
[2024-02-07T08:57:15.224+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:57:15.224+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:57:15.227+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:57:15.226+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/10_emp_details.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/10_emp_details.py", line 19, in <module>
    import Task_builder
ModuleNotFoundError: No module named 'Task_builder'
[2024-02-07T08:57:15.227+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:57:15.245+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/10_emp_details.py took 0.027 seconds
[2024-02-07T08:57:45.402+0000] {processor.py:161} INFO - Started process (PID=1794) to work on /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:57:45.404+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/10_emp_details.py for tasks to queue
[2024-02-07T08:57:45.405+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:57:45.405+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:57:45.409+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:57:45.409+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/10_emp_details.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/10_emp_details.py", line 19, in <module>
    import Task_builder
ModuleNotFoundError: No module named 'Task_builder'
[2024-02-07T08:57:45.410+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:57:45.428+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/10_emp_details.py took 0.029 seconds
[2024-02-07T08:58:16.414+0000] {processor.py:161} INFO - Started process (PID=1860) to work on /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:58:16.415+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/10_emp_details.py for tasks to queue
[2024-02-07T08:58:16.417+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:58:16.416+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:58:16.420+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:58:16.419+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/10_emp_details.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/10_emp_details.py", line 19, in <module>
    import Task_builder
ModuleNotFoundError: No module named 'Task_builder'
[2024-02-07T08:58:16.420+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:58:16.438+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/10_emp_details.py took 0.026 seconds
[2024-02-07T08:58:47.225+0000] {processor.py:161} INFO - Started process (PID=1926) to work on /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:58:47.226+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/10_emp_details.py for tasks to queue
[2024-02-07T08:58:47.228+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:58:47.228+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:58:47.232+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:58:47.230+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/10_emp_details.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/10_emp_details.py", line 19, in <module>
    import Task_builder
ModuleNotFoundError: No module named 'Task_builder'
[2024-02-07T08:58:47.232+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:58:47.252+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/10_emp_details.py took 0.030 seconds
[2024-02-07T08:59:18.218+0000] {processor.py:161} INFO - Started process (PID=1992) to work on /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:59:18.219+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/10_emp_details.py for tasks to queue
[2024-02-07T08:59:18.222+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:59:18.221+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:59:18.225+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:59:18.225+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/10_emp_details.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/10_emp_details.py", line 19, in <module>
    import Task_builder
ModuleNotFoundError: No module named 'Task_builder'
[2024-02-07T08:59:18.226+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:59:18.245+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/10_emp_details.py took 0.029 seconds
[2024-02-07T08:59:49.091+0000] {processor.py:161} INFO - Started process (PID=2058) to work on /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:59:49.092+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/10_emp_details.py for tasks to queue
[2024-02-07T08:59:49.094+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:59:49.094+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:59:49.097+0000] {logging_mixin.py:188} INFO - [2024-02-07T08:59:49.096+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/10_emp_details.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/10_emp_details.py", line 19, in <module>
    import Task_builder
ModuleNotFoundError: No module named 'Task_builder'
[2024-02-07T08:59:49.097+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/10_emp_details.py
[2024-02-07T08:59:49.117+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/10_emp_details.py took 0.029 seconds
[2024-02-07T09:00:19.199+0000] {processor.py:161} INFO - Started process (PID=2124) to work on /opt/airflow/dags/10_emp_details.py
[2024-02-07T09:00:19.201+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/10_emp_details.py for tasks to queue
[2024-02-07T09:00:19.202+0000] {logging_mixin.py:188} INFO - [2024-02-07T09:00:19.202+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/10_emp_details.py
[2024-02-07T09:00:19.206+0000] {logging_mixin.py:188} INFO - [2024-02-07T09:00:19.205+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/10_emp_details.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/10_emp_details.py", line 19, in <module>
    import Task_builder
ModuleNotFoundError: No module named 'Task_builder'
[2024-02-07T09:00:19.206+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/10_emp_details.py
[2024-02-07T09:00:19.228+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/10_emp_details.py took 0.032 seconds
[2024-02-07T09:00:49.497+0000] {processor.py:161} INFO - Started process (PID=2190) to work on /opt/airflow/dags/10_emp_details.py
[2024-02-07T09:00:49.499+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/10_emp_details.py for tasks to queue
[2024-02-07T09:00:49.501+0000] {logging_mixin.py:188} INFO - [2024-02-07T09:00:49.500+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/10_emp_details.py
[2024-02-07T09:00:49.506+0000] {logging_mixin.py:188} INFO - [2024-02-07T09:00:49.505+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/10_emp_details.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/10_emp_details.py", line 19, in <module>
    import Task_builder
ModuleNotFoundError: No module named 'Task_builder'
[2024-02-07T09:00:49.507+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/10_emp_details.py
[2024-02-07T09:00:49.530+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/10_emp_details.py took 0.037 seconds
[2024-02-07T09:01:20.517+0000] {processor.py:161} INFO - Started process (PID=2256) to work on /opt/airflow/dags/10_emp_details.py
[2024-02-07T09:01:20.519+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/10_emp_details.py for tasks to queue
[2024-02-07T09:01:20.520+0000] {logging_mixin.py:188} INFO - [2024-02-07T09:01:20.520+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/10_emp_details.py
[2024-02-07T09:01:20.523+0000] {logging_mixin.py:188} INFO - [2024-02-07T09:01:20.523+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/10_emp_details.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/10_emp_details.py", line 19, in <module>
    import Task_builder
ModuleNotFoundError: No module named 'Task_builder'
[2024-02-07T09:01:20.524+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/10_emp_details.py
[2024-02-07T09:01:20.542+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/10_emp_details.py took 0.027 seconds
[2024-02-07T09:01:51.385+0000] {processor.py:161} INFO - Started process (PID=2322) to work on /opt/airflow/dags/10_emp_details.py
[2024-02-07T09:01:51.386+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/10_emp_details.py for tasks to queue
[2024-02-07T09:01:51.388+0000] {logging_mixin.py:188} INFO - [2024-02-07T09:01:51.387+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/10_emp_details.py
[2024-02-07T09:01:51.391+0000] {logging_mixin.py:188} INFO - [2024-02-07T09:01:51.391+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/10_emp_details.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/10_emp_details.py", line 19, in <module>
    import Task_builder
ModuleNotFoundError: No module named 'Task_builder'
[2024-02-07T09:01:51.392+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/10_emp_details.py
[2024-02-07T09:01:51.411+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/10_emp_details.py took 0.030 seconds
[2024-02-07T09:02:21.584+0000] {processor.py:161} INFO - Started process (PID=2388) to work on /opt/airflow/dags/10_emp_details.py
[2024-02-07T09:02:21.585+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/10_emp_details.py for tasks to queue
[2024-02-07T09:02:21.587+0000] {logging_mixin.py:188} INFO - [2024-02-07T09:02:21.586+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/10_emp_details.py
[2024-02-07T09:02:21.589+0000] {logging_mixin.py:188} INFO - [2024-02-07T09:02:21.589+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/10_emp_details.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/10_emp_details.py", line 19, in <module>
    import Task_builder
ModuleNotFoundError: No module named 'Task_builder'
[2024-02-07T09:02:21.590+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/10_emp_details.py
[2024-02-07T09:02:21.610+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/10_emp_details.py took 0.029 seconds
[2024-02-07T09:02:52.363+0000] {processor.py:161} INFO - Started process (PID=2454) to work on /opt/airflow/dags/10_emp_details.py
[2024-02-07T09:02:52.364+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/10_emp_details.py for tasks to queue
[2024-02-07T09:02:52.365+0000] {logging_mixin.py:188} INFO - [2024-02-07T09:02:52.365+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/10_emp_details.py
[2024-02-07T09:02:52.369+0000] {logging_mixin.py:188} INFO - [2024-02-07T09:02:52.368+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/10_emp_details.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/10_emp_details.py", line 19, in <module>
    import Task_builder
ModuleNotFoundError: No module named 'Task_builder'
[2024-02-07T09:02:52.369+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/10_emp_details.py
[2024-02-07T09:02:52.389+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/10_emp_details.py took 0.029 seconds
[2024-02-07T09:03:23.386+0000] {processor.py:161} INFO - Started process (PID=2526) to work on /opt/airflow/dags/10_emp_details.py
[2024-02-07T09:03:23.387+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/10_emp_details.py for tasks to queue
[2024-02-07T09:03:23.389+0000] {logging_mixin.py:188} INFO - [2024-02-07T09:03:23.389+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/10_emp_details.py
[2024-02-07T09:03:23.392+0000] {logging_mixin.py:188} INFO - [2024-02-07T09:03:23.391+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/10_emp_details.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/10_emp_details.py", line 19, in <module>
    import Task_builder
ModuleNotFoundError: No module named 'Task_builder'
[2024-02-07T09:03:23.392+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/10_emp_details.py
[2024-02-07T09:03:23.411+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/10_emp_details.py took 0.027 seconds
[2024-02-07T09:03:54.319+0000] {processor.py:161} INFO - Started process (PID=2592) to work on /opt/airflow/dags/10_emp_details.py
[2024-02-07T09:03:54.321+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/10_emp_details.py for tasks to queue
[2024-02-07T09:03:54.322+0000] {logging_mixin.py:188} INFO - [2024-02-07T09:03:54.322+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/10_emp_details.py
[2024-02-07T09:03:54.325+0000] {logging_mixin.py:188} INFO - [2024-02-07T09:03:54.324+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/10_emp_details.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/10_emp_details.py", line 19, in <module>
    import Task_builder
ModuleNotFoundError: No module named 'Task_builder'
[2024-02-07T09:03:54.325+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/10_emp_details.py
[2024-02-07T09:03:54.343+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/10_emp_details.py took 0.027 seconds
